src/engine/cachingreader/cachingreaderworker.cpp

```diff
@@ -68,6 +69,17 @@ ReaderStatusUpdate CachingReaderWorker::processReadRequest(
         }
     }
 
+    // This call here assumes that the caching reader will read the first sound cue at
+    // one of the first chunks. When it comes along, it verifies it as a sanity check
+    // for unchanged track samples and decoder.
+    // This is part of a first prove of concept and needs to be replaces with an
+    // explicit lookup and result handling designed for this purpose.
+    // This lookup should happen, before anything else is read. This might be extended
+    // to further checks whether a automatic offset adjustment is possible or a the
+    // sample position metadata shall be treated as outdated.
+    // Currently we log this case in the mixxx.log only.
```

Comment:
Since we have not yet agreed to a alternative solution, I have reword it again as "solution which is still under discussion"
I hope that is least controversial.

Reply:
What sentence exactly do you intend to reword?

Reply:
Can we at least document that the verification should not stay in `CachingReader`?